casbin/internal_enforcer.py

```diff
@@ -88,6 +88,28 @@ def _update_policies(self, sec, ptype, old_rules, new_rules):
 
         return rules_updated
 
+    def _update_filtered_policies(
+        self, sec, ptype, new_rules, field_index, *field_values
+    ):
+        """updates rules from currect policy on the basis of filter"""
+
+        if self.adapter and self.auto_save:
+
+            if (
+                self.adapter.update_filtered_policies(
+                    sec, ptype, new_rules, field_index, *field_values
+                )
+                is False
+            ):
+                return False
+
```

Comment:
Here you need to update the policy in the model.
Also please format the code to remove unnecessary blank lines and line breaks.

Reply:
@leeqvip can you review it again I have made changes